### PR TITLE
style: include spaces around all variables

### PR DESCRIPTION
**Pull request by gclough**

### CODE REVIEW

File: tasks/users_privileges.yml [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
-    login_host: "{{item.host | default(omit)}}"
-    login_user: "{{postgresql_admin_user}}"
-    role_attr_flags: "{{item.role_attr_flags | default(omit)}}"
+    login_host: "{{ item.host | default(omit) }}"
+    login_user: "{{ postgresql_admin_user }}"
+    role_attr_flags: "{{ item.role_attr_flags | default(omit) }}"
   become: yes
-  become_user: "{{postgresql_admin_user}}"
-  with_items: "{{postgresql_user_privileges}}"
+  become_user: "{{ ostgresql_admin_user }}"
```
Comment (gclough) [inline tasks/users_privileges.yml:14]: Typo

**Comment by MrMegaNova** [comment]
I have no idea, I've just retry the failed check, we'll see
I have no time to debug today, I will take a look tomorrow or next week

Please do not push or request direct push to master anymore, fist create a PR to development then when development branch are stable merge into master

**Comment by MrMegaNova** [comment]
Thanks for your work

**Comment by gclough** [comment]
Glad to help out @MrMegaNova, as I've been in your shoes... years ago when it was just me maintaining it.  :-(

Also, can we change from `master` -> `main` at some point?

**Comment by MrMegaNova** [comment]
> Glad to help out @MrMegaNova, as I've been in your shoes... years ago when it was just me maintaining it. :-(
> 
> Also, can we change from `master` -> `main` at some point?

Happy to see you back on this project, hoping our latest updates will bring back some maintainers on this project
Great idea, maybe next week because it require some changes to rename the master branch into main. You can open an issue about this to keep it in mind.

**Pull request closed without merging**